src/pages/index.js

```diff
@@ -19,18 +20,79 @@ import {
   addNewCardButton,
   profileAddCardForm,
   cardsWrap,
+  avatarEditButton,
 } from "../utils/constants.js";
 
 /* -------------------------------------------------------------------------- */
 /*                                  Functions                                 */
 /* -------------------------------------------------------------------------- */
+const BACKEND_URL = "https://around-api.en.tripleten-services.com/v1";
+//API
+
+const api = new Api({
+  baseUrl: BACKEND_URL,
+  headers: {
+    authorization: "bf54a6ce-ba6a-4742-a5e8-9bdd0fb7708a",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardSection;
+let userId;
 
 // User information
-export const userInfo = new UserInfo(
+const userInfo = new UserInfo(
   document.querySelector(".profile__title"),
-  document.querySelector(".profile__description")
+  document.querySelector(".profile__description"),
+  document.querySelector(".profile__image")
 );
 
+// User Info
+Promise.all([api.getUserInfo()])
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all` because it might happen so that the `_id` has not been received yet but the cards have and your cards will be rendered not correctly. 
More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```